### ISSUE TITLE
add additional features to colorize-svgs.sh

### ISSUE THREAD

**Issue by aruguy** [game-icons/icons#990]
Would it be possible to add additional features like defining shadows or strokes to the shell script `colorize-svgs.sh`?
I would like to create a repository of icons using the same color paths, but your shell script only seems to change background/foreground, and I would like to add a foreground shadow to match the icon style listed below.

1. Set background shape to square
2. Change background gradient colour to #1B1F21
3. Change foreground gradient colour to #E64539
4. Enable foreground shadow
5. Set foreground shadow to side in with blur size 15
6. Change foreground shadow colour to #3A3F5E
7. Download as .svg

**Comment by Delapouite** [comment]
Yes, the goal is to enhance this script with features you've described.
